PauseIndexing.ps1

```diff
@@ -0,0 +1,82 @@
+﻿[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$True, Position=0, HelpMessage="The Server Instance against which the script is to run.")]
+    [string]$SQLServerInstance,
+
+    [Parameter(Mandatory=$True, Position=1, HelpMessage="Configuration DB")]
+    [string]$ConfigurationDatabaseName,
+    
+    [Parameter(Mandatory=$False, Position=2, HelpMessage="Pause Indexing for Code, WorkItem or All.")]
+    [string]$EntityType = "All"

```

Comment:
Switches might be easier to use `PauseIndexing.ps1 -ServerInstance foo -ConfigurationDabatabase -All`